distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc

```diff
@@ -0,0 +1,76 @@
+:title: System Configuration Settings
+:type: installing
+:status: published
+:summary: Configure Single Sign On(SSO) page of installer.
+:project: ${branding}
+:order: 07
+
+.[[_configuring_sso]]Configure Single Sign On(SSO)
```

Comment:
❓ There should be a space before `(SSO)`, right?

Reply:
fixed